src/oc/hoc.cpp

```diff
@@ -960,16 +958,10 @@ void hoc_final_exit(void) {
     rl_deprep_terminal();
 #endif
     ivoc_cleanup();
-#ifdef WIN32
-    hoc_win32_cleanup();
-#else
-    std::string cmd{neuron_home};
-    cmd += "/lib/cleanup ";
-    cmd += std::to_string(hoc_pid());
-    if (system(cmd.c_str())) {  // fix warning: ignoring return value
-        return;
-    }
-#endif
+    auto tmp_dir = std::getenv("TEMP");
+    auto path = std::filesystem::path(tmp_dir ? std::string(tmp_dir) : "/tmp") /
+                ("oc" + std::to_string(hoc_pid()) + ".hl");
```

Comment:
```suggestion
                fmt::format("oc{}.hl", hoc_pid());
```

Reply:
As a side note, I'm a bit annoyed there's no way (that I know of, I'd be happy to be proven wrong!) to have Python-style f-strings, either in `fmt` or in C++20. Closest thing that works is:

```cpp
using namespace fmt::literals;
fmt::format("oc{pid}.hl", "pid"_a = hoc_pid());
```

but the double repetition of `pid` (in the string itself and as an arg) is an eyesore. Meanwhile, with `{}` I need to remember to backtrack to the string to see if the items are inserted in the right place...

/endrant